app/controllers/api/v0/users_controller.rb

```diff
@@ -17,10 +17,16 @@ def show_user_by_id
   def show_users_by_id
     user_ids = params.require(:ids)
     users = User.where(id: user_ids)
+    private_attributes = []
+    if current_user && current_user.can_admin_results?
+      # These attributes are needed to check anonymization status during anonymization.
+      private_attributes = %w[dob email]
+    end
```

Comment:
I'm not too wild about the way you're "hacking" confidential data into this endpoint. Where is it being called?